src/main/java/io/axoniq/axonserver/connector/admin/LoadBalanceStrategy.java

```diff
@@ -0,0 +1,37 @@
+package io.axoniq.axonserver.connector.admin;
+
+/**
+ * @author Stefan Dragisic
```

Comment:
The JavaDoc of this class is incomplete.
I expect the following still:

* A class level description
* A description per enumeration option
* A description for *any* public method